platform/web-security/src/test/java/org/motechproject/security/service/MotechUserServiceTest.java

```diff
@@ -67,4 +77,21 @@ public void shouldValidatePasswordOnEdit() {
 
         motechUserService.updateUserDetailsWithPassword(userDto);
     }
+
+    @Test(expected = LockedException.class)
+    public void shouldBlockUserAfterCrossingTheFailureLoginCounter() {
```

Comment:
We should also have a test that checks that method increments the failed logins counter properly